mobile_app_switzerland/controllers/registration_controller.py

```diff
@@ -20,4 +21,5 @@ def registration_form(self, **kwargs):
         """
         Return registration form
         """
-        return request.redirect('compassionappuk://')
+        hostname = config.get("wordpress_host")
+        return request.redirect('https://' + hostname + '/Login')
```

Comment:
I think we wanted to put every mobile page inside the same path. That's what I changed on iOS:
```suggestion
        return request.redirect('https://' + hostname + '/app/Login')
```

Reply:
Good catch!